src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -28,23 +31,10 @@
     public String notificationChannel;
 
     /**
-     * Pending, unverified phone number, in +15551234 format (E.164 format, includes country code, no spaces)
-     * and for which a verification code has been requested.
-     */
-    public String pendingPhoneNumber;
-    /**
-     * Locally-formatted pendingPhoneNumber for display.
-     */
-    public String pendingPhoneNumberFormatted;
-
-    /**
-     * Verified phone number for SMS notifications, in +15551234 format (E.164 format, includes country code, no spaces).
+     * Phone number for SMS notifications.
+     * Must include country code, e.g., +15551234
      */
     public String phoneNumber;
-    /**
-     * Locally-formatted phoneNumber for display.
-     */
-    public String phoneNumberFormatted;
```

Comment:
Thanks for removing. If we want some special formatting, we should just handle on the client side with an npm lib.